README.md

```diff
@@ -23,7 +25,10 @@ Latest image: `gcr.io/google-containers/volume-csi/gcp-compute-persistent-disk-c
 This plugin is compatible with CSI versions [v0.2.0](https://github.com/container-storage-interface/spec/blob/v0.2.0/spec.md) and [v0.3.0](https://github.com/container-storage-interface/spec/blob/v0.3.0/spec.md)
 
 ### Kubernetes Compatibility
-This plugin can be used as-is beginning with Kubernetes v1.10.5
+| GCE PD CSI Driver\Kubernetes Version | <1.10.4 | 1.10.x where x>4 | 1.11 | 1.12+ |
```

Comment:
I would make this table simpler and just say
v0.1.0.alpha: 1.10.5 - 1.11
latest: 1.12+